Automattic/wp-calypso
# Performance Profiler: updates loading screen.

Fixes https://github.com/Automattic/dotcom-forge/issues/8693

## Proposed Changes

Add the new loading screen from Figma i2.
Update to the next step every 5s and to another random tip every 10s.



## Testing Instructions



- Go to http://calypso.localhost:3000/speed-test-tool?url=https%3A%2F%2Fwww.quintoandar.com.br%2F&tab=mobile
- Check that loading screen appears
- Every 5 secs the step should change 
- Every 10s a random tip should be shown
- After 20-30 secs the dashboard content should be revealed

https://github.com/user-attachments/assets/2741612e-dc22-4bba-9def-8984d53937f4


## Pre-merge Checklist
